lib/src/features/line_renderer.dart

```diff
@@ -47,6 +49,12 @@ class LineRenderer extends FeatureRenderer {
     effectivePaint.strokeWidth =
         context.tileSpaceMapper.widthFromPixelToTile(strokeWidth);
 
+    var dashLengths = style.lineLayout!.dashArray;
```

Comment:
consider `style.lineLayout?` instead so that we don't get an exception here if there is no layout

Reply:
done.